hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmAcls.java

```diff
@@ -204,6 +208,58 @@ public void testSetACLPermissionDenied() throws Exception {
     verifyAuditLog(OMAction.SET_ACL, AuditEventStatus.FAILURE);
   }
 
+  @Test
+  public void testSetlACLKeyPermissionDenied() throws Exception {
```

Comment:
We can rename this test with `testKeyACLOpsPermissionDenied` (or something similar) since it includes other ACL ops (GET, SET, ADD, REMOVE).

Reply:
There is also a typo in the current name, so let's rename as @ivandika3 suggested.

```suggestion
  public void testKeyACLOpsPermissionDenied() throws Exception {
```